.gitignore

```diff
@@ -26,6 +26,7 @@ __pycache__
 .coverage
 *.swp
 .pytest_cache
+.hypothesis
```

Comment:
You should think of adding this to [your global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)